sycl/test-e2e/USM/fill_any_size.cpp

```diff
@@ -0,0 +1,81 @@
+// RUN: %{build} -o %t1.out
+// RUN: %{run} %t1.out
+// XFAIL: (opencl && cpu)
+// XFAIL-TRACKER: https://github.com/oneapi-src/unified-runtime/issues/2440
```

Comment:
The issue says

> confirmed that it's the clEnqueueMemFillINTEL call which sometimes returns -30 (CL_INVALID_VALUE).

meaning the failure is flaky and sometimes the test passes. How are you marking it with XFAIL then? We're also seeing "Unexpected Passes" occasionally in downstream now (at least, maybe here as well).